app/src/main/java/org/xbmc/kore/utils/Utils.java

```diff
@@ -250,4 +253,28 @@ public void onError(int errorCode, String description) {
             }
         }, callbackHandler);
     }
+
+    public static void setLocale(Context context, String localeName) {
+        Locale locale = getLocale(localeName);
+
+        Locale.setDefault(locale);
+
+        Resources resources = context.getResources();
+
+        Configuration configuration = resources.getConfiguration();
+        configuration.locale = locale;
+
+        resources.updateConfiguration(configuration, resources.getDisplayMetrics());
+    }
+
+    public static Locale getLocale(String localeName) {
+        Locale locale;
+        String[] languageAndRegion = localeName.split("_", 2);
```

Comment:
Ok, so this is where the `_` comes from...

Don't think it's a good idea to use `_`, when the locale specification uses `-`.

Reply:
It's `_`. see `BuildConfig.SUPPORTED_LOCALES`

Reply:
The `_` is something that you defined, it's not standard. The standard is `-`, so that's what should be used. `BuildConfig.SUPPORTED_LOCALES` should be changed also.

Reply:
Dug into it a little deeper and you're right, IETF seems to be the standard way to do it and uses the hyphen as separator.